### ISSUE TITLE
Unable to use custom URL paths due to port parsing

### ISSUE THREAD

**Issue by MysticRyuujin** [kvhnuke/etherwallet#1261]
Currently MEW requires us to enter the port number in a separate input box and then does a string concatenation of url + ":" + port

However, this makes it impossible to use an RPC URI that isn't root level E.G.

`http://ethereum-harmony:8080/rpc/`

Would it be difficult to just allow us to enter the URI normally, like above?

I've requested/posted to Ethereum Harmony a request to allow the RPC port to be at the root of 8545 but I still think this should be discussed and/or an available option in MEW 😄

**Comment by tayvano** [comment]
Would making the port number optional work? That way you could input `http://ethereum-harmony:8080/rpc/` and nothing? It's a pretty simple change. 

If you have the ability to test it, let me know. I'm short on time these days and that would be helpful. Let me know. Thanks!

**Comment by MysticRyuujin** [comment]
@tayvano yeah, if the port input is optional that would be fine and yes I can test it, no problem 👍

**Comment by MysticRyuujin** [comment]
So I wasn't sure if you were asking me to test this before a patch? Anyway it does not work if I enter:
http://node:8080/rpc/

Using the dev/debug tools in chrome/firefox shows no connection attempt in the browser to the configured custom node. It seems to accept my config but it just says:

`Could not connect to the node. Refresh your page, try a different node (upper right corner), check your firewall settings. If custom node, check your configs. `

Also, #924 makes testing this annoying 😄
